section4/exercises/classMethods.js

```diff
@@ -28,13 +28,32 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
-
   // ADD CODE
+  changeProtein() {
+    this.protein = "Duck";
+  }
+  addTopping() {
+    console.log("Guacamole");
```

Comment:
You may need to research how to add an element to an array to figure out this one- your toppings attribute is an array, so this method should be able to put another element in that array. Again, consider using a parameter!